easybuild/easyconfigs/i/IPM/darshan-mk-log-dirs.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/perl -w
```

Comment:
`darshan-mk-log-dirs.pl` is used to create the tree of dirs for archiving reports of all users in 1 shared dir, it should be run manually (depends on each site policy).